package.json

```diff
@@ -12,12 +12,11 @@
   ],
   "scripts": {
     "test": "npm run test:code && npm run test:tsc && npm run test:tsd && npm run lint",
-    "test:code": "c8 --100 ava test/*.test.js",
```

Comment:
I'm bummed we loos the 100% test coverage. Do you see any way to bring that back? I think Node's test runner supports coverage but probably not back to Node 16? I don't mind releasing a breaking version and drop support for everything older than Node 20 if that helps

Reply:
coverage support is afaik availabe since node 22. So I brought back c8.

Reply:
thanks!